features/breadcrumbs.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Leaving breadcrumbs to attach to reports
```

Comment:
There are several undefined steps in these scenarios, which can be seen in the [CI job](https://travis-ci.com/bugsnag/bugsnag-unity/jobs/155575507). Therefore these scenarios do not currently assert that breadcrumbs are added to error reports.

Reply:
Additionally, attempting to run mazerunner locally after following the contribution instructions results in the following error message:

>rake aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - unity/PackageProject/Assets/Plugins/OSX/Bugsnag/bugsnag-osx.bundle/Contents/_CodeSignature/CodeResources.meta
/Users/jamielynch/repos/bugsnag-unity/Rakefile:234:in `initialize'
/Users/jamielynch/repos/bugsnag-unity/Rakefile:234:in `open'
/Users/jamielynch/repos/bugsnag-unity/Rakefile:234:in `block (5 levels) in <top (required)>'
/Users/jamielynch/repos/bugsnag-unity/Rakefile:233:in `each_line'
/Users/jamielynch/repos/bugsnag-unity/Rakefile:233:in `block (4 levels) in <top (required)>'
/Users/jamielynch/repos/bugsnag-unity/Rakefile:232:in `block (3 levels) in <top (required)>'
/Users/jamielynch/repos/bugsnag-unity/Rakefile:231:in `block (2 levels) in <top (required)>'
/usr/local/lib/ruby/gems/2.5.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => plugin:maze_runner => plugin:export => plugin:check_package
(See full trace by running task with --trace)